Source/BluemixAppID/internal/LoginWidgetImpl.swift

```diff
@@ -21,7 +21,11 @@ public class LoginWidgetImpl: LoginWidget {
     }
     
     public func launch(delegate: AuthorizationDelegate) {
-        self.oauthManager.authorizationManager?.launchAuthorizationUI(authorizationDelegate: delegate)
+        launch(accessTokenString: nil, delegate: delegate)
+    }
+    
+    public func launch(accessTokenString: String?, delegate: AuthorizationDelegate) {
+        self.oauthManager.authorizationManager?.launchAuthorizationUI(accessTokenString: accessTokenString, authorizationDelegate: delegate)
     }
```

Comment:
having one method signature with optional arguments with default value should be enough